### ISSUE TITLE
No longer actively maintained repositories

### ISSUE THREAD

**Issue by chaconnewu** [sindresorhus/awesome#628]
According to our discussion https://github.com/sindresorhus/awesome/pull/626, I generated a list of not-pushed-in-previous-3-month repositories with date pulling from GitHub API:

-   [Cordova](https://github.com/busterc/awesome-cordova)
-   [Dart](https://github.com/yissachar/awesome-dart)
-   [Coldfusion](https://github.com/seancoyne/awesome-coldfusion)
-   [Fortran](https://github.com/rabbiabram/awesome-fortran)
-   [Web Components](https://github.com/mateusortiz/webcomponents-the-right-way)
-   [Dojo Toolkit](https://github.com/peterkokot/awesome-dojo)
-   [Inspiration](https://github.com/NoahBuscher/Inspire)
-   [Play1 Framework](https://github.com/PerfectCarl/awesome-play1)
-   [Speech and Natural Language Processing](https://github.com/edobashira/speech-language-processing)
    -   [Spanish](https://github.com/dav009/awesome-spanish-nlp)
-   [Spanish](https://github.com/dav009/awesome-spanish-nlp)
-   [Linguistics](https://github.com/theimpossibleastronaut/awesome-linguistics)
-   [Cryptography](https://github.com/MaciejCzyzewski/retter)
-   [Dev Env](https://github.com/jondot/awesome-devenv)
-   [Speaking](https://github.com/matteofigus/awesome-speaking)
-   [Tech Videos](https://github.com/lucasviola/awesome-tech-videos)
-   [Radio](https://github.com/kyleterry/awesome-radio)
-   [Open Companies](https://github.com/opencompany/awesome-open-company)
-   [Services Engineering](https://github.com/mmcgrana/services-engineering)
-   [Sketch](https://github.com/diessica/awesome-sketch) *(OS X app)*
-   [FOSS Production Apps](https://github.com/jwaterfaucett/awesome-foss-apps)
-   [Answers](https://github.com/stoeffel/awesome-ama-answers)
-   [Productivity](https://github.com/jyguyomarch/awesome-productivity)
-   [JSON Datasets](https://github.com/jdorfman/awesome-json-datasets)

**Comment by sindresorhus** [comment]
@chaconnewu I don't think 3 months is a good enough metric though. How many open PRs is also a good factor. Some lists just aren't that active just because the community is small or inactive.

**Comment by davisonio** [comment]
I think we're mostly concerned about the lists where people's contributions (pull-requests) are not being reviewed, I've gone through your list and here's the ones where I think there's a bit of a problem with inactivity:
-   [Cordova](https://github.com/busterc/awesome-cordova) - 1 unreviewed PR
-   [Play1 Framework](https://github.com/PerfectCarl/awesome-play1) - 2 unreviewed PRs
-   [Speech and Natural Language Processing](https://github.com/edobashira/speech-language-processing) - 10 unreviewed PRs
-   [Dev Env](https://github.com/jondot/awesome-devenv) - 5 unreviewed PRs

**Comment by chaconnewu** [comment]
@davisonio @sindresorhus I think that the number of open PRs is definitely one important perspective. So maybe it is better to use different metrics for bigger and smaller community?

**Comment by davisonio** [comment]
> So maybe it is better to use different metrics for bigger and smaller community?

Not necessarily, I think that mostly the awesome lists need to be monitored manually regularly to check that people's contributions are being taken into account.

The most useful thing to generate in my opinion would be to check for unreviewed pull requests.

---

@busterc @PerfectCarl @edobashira @jondot

Are you able to review the pull requests for the awesome lists highlighted above /  add a collaborator to do so on your behalf?

**Comment by busterc** [comment]
@davisonio I agree that unreviewed PRs are the better metric to consider. Also, I've reviewed the 1 outstanding PR and updating the list.

If anybody out there is interested in being a collaborator on [Awesome Cordova](https://github.com/busterc/awesome-cordova), just let me know (add an issue).

**Comment by aleksandar-todorovic** [comment]
Maybe instead of "nothing pushed in the last three months" we can have something like "no pull requests reviewed in the last three months"?

**Comment by chaconnewu** [comment]
In my opinion, 'no pull requests reviewed' partially reflects the maintainer's commitment. It signals the owner's willingness to interact with the community. However, what if authors don't hold the same standard and just try to comply with this metric? I'm more concerned about the maintenance of existing resources. Many curated lists index software projects, and many indexed projects are deprecated if they are no long maintained or replaced by better ones, which should be reflected in the lists.

**Comment by rachmadaniHaryono** [comment]
what about migrated project? https://github.com/rosarior/awesome-django is migrated to gitlab and didn't process any PR from december

**Comment by JesseTG** [comment]
Why not add a `.travis.yml` to this project that, among other things, checks to see which repos are active?  It doesn't even have to cause a failure, it could just e-mail @sindresorhus or post an automatic issue here or something.

**Comment by sindresorhus** [comment]
@JesseTG How do you programmatically define inactivity though? Some lists are just inactive because they're about an obscure thing. Some are inactive because they're pretty much done.

**Comment by JesseTG** [comment]
@sindresorhus Maybe once every six months or so the maintainer should check in on an issue thread or with an e-mail or something, if they don't have a reason to update the list itself?

**Comment by web-padawan** [comment]
I know that this issue is outdated, would like to let you know that https://github.com/mateusortiz/webcomponents-the-right-way is maintained again. I have put some effort into actualising that list, and will be working to keep it up-to-date.

**Comment by jjangga0214** [comment]
@JesseTG @sindresorhus I got what @sindresorhus worried about. Yes, there is no simple metric to carefully decide which repo is inactive. However, I think @JesseTG's idea is also great, not to final decision making, but to suggesting some repos that we 'might' reconsider.

**Comment by davisonio** [comment]
@chaconnewu wonder if you could share the script you used to calculate these back in 2017? Would be great to run this again to see if there are any more unmaintained lists.

**Comment by elenigen** [comment]
There's [graphql](https://github.com/chentsulin/awesome-graphql) which has 79 PRs and almost two months of inactivity.
Is there something we can do? ... Fork and change the reference?

**Comment by davisonio** [comment]
Try emailing the creator first @chentsulin
I'm happy to fork and maintain if needs be but two months isn't very long for inactivity let's wait first

**Comment by elenigen** [comment]
I sent an email, so let's see if it will unblock, now it's been 3 months.

**Comment by elenigen** [comment]
@davisonio I sent an email to @chentsulin on Jan 20, 2020 and still no reply. We have now 112 PRs, last activity on Feb 12 and people agree with me that it should be faster to review and merge PRs, so we need more admins but for some reason they don't grant admin to others.
https://github.com/chentsulin/awesome-graphql/issues/636

**Comment by sancarn** [comment]
> @davisonio I sent an email to @chentsulin on Jan 20, 2020 and still no reply. We have now 112 PRs, last activity on Feb 12 and people agree with me that it should be faster to review and merge PRs, so we need more admins but for some reason they don't grant admin to others. [chentsulin/awesome-graphql#636](https://github.com/chentsulin/awesome-graphql/issues/636)

Half feel like there should be an `awesome` organisation which owns repos like this, and the `awesome` repo itself and have the power to merge PRs in all repos. I can totally understand being in a situation where you are burnt out and no longer want to maintain something if you don't use the technology anymore, and don't get paid for either.

**Comment by Bongdautay** [comment]
quaroibn1@gmail.com

**Comment by j2346391** [comment]
wait a second. you probably misunderstand something.

can somebody talk to me?

Bongdautay ***@***.***>於 2024年9月6日 週五，上午2:06寫道：

**Comment by Coco197** [comment]
There is someone who uses my account to play Xbox One for free and I have been paying for it for a year to avoid the extra revenue it costs for Microsoft. And for a year I paid and I am unemployed and during that time I had to stop paying another account to pay for x box. I don't want to file a police report because I hope the people behind this will give me my money back. My email is joaquinmiranda@hotmail.com I expect a response soon and if I had a job I wouldn't mind because I would have a lot more money. I have been unemployed for 8 years and I only have help from some projects from the unemployed fund, I don't know but it is unfair that I am paying for a luxury that I don't want and that I have never asked for and for a while I thought about filing a complaint against the police but I don't think they use the police for this I think people still made a mistake and the person can give me the money

**Comment by DixieBarrett** [comment]
I have used the services of this great Spytechacker that has helped a lot of students to conquer their fears as a result of poor grades, a lot of students have issues but don’t know who or whom to turn to.
Before now i was in this same situation but when i hired hackerspytech @ gmail com i was able to navigate the crazy grading of my college academic score. ask about imap for british students and Imat for italian students
i can only encourage you to make use of their service to help you with your school grades or any other hacking services.

**Comment by amirrh6** [comment]
The repository https://github.com/yissachar/awesome-dart was last updated 2 years ago, and https://github.com/dypsilon/frontend-dev-bookmarks hasn’t been updated in 7 years!!!

These links might be considered obsolete, so it could be helpful to add a warning note next to their links in the awesome list.

**Comment by Angel-Mathew007** [comment]
Hi!
I’m interested in contributing to this project and would like to help with this issue. Could you please provide some guidance on how best to proceed or assign it to me? I’m excited to contribute and learn.

Thank you!

**Comment by dertuxmalwieder** [comment]
The Linux "awesome list" has been archived in 2023. Remove or add a note?
